roles/swift-ring/tasks/main.yml

```diff
@@ -1,15 +1,8 @@
 ---
-- name: register whether we need to boostrap rings
-  shell: /usr/bin/test -f /etc/swift/account.ring.gz || /usr/bin/test -f /etc/swift/container.ring.gz || /usr/bin/test -f /etc/swift/object.ring.gz
-  failed_when: False
-  register: should_boostrap
-
 - name: drop our ring configuration
   copy: src={{ swift_ring.ring_definition_file }} owner=root group=root
         dest=/etc/swift/ring_definition.yml mode=644 backup=yes
-  when: not should_boostrap|success
 
 - name: setup swift rings
```

Comment:
should we set this to be a handler and to be notified by the "drop our ring configuration" task ?   That way it only runs when the ring config file changes...